src/app/feature-flags-settings/feature-flags-settings.component.html

```diff
@@ -1,5 +1,14 @@
 <div id="setting-item">
     <app-loader *ngIf="loading"> </app-loader>
+    <div class="container" *ngIf="!loading && !authorizationParameters">
+        <configcat-authorization (error)="error()" (authorized)="login($event)"></configcat-authorization>
+    </div>
+    <div class="container" *ngIf="!loading && authorizationParameters && showError">
+        <div class="error">
+            <p><b>Something went wrong!</b></p>
+            <p>Please try again later or for more inforamtion check the console logs.</p>
```

Comment:
```suggestion
            <p>You can try again later or check the console log for more information.</p>
```

B version:

You can try again later, check the console log for more information or contact us [link].

Not sure if this is applicable here.

Reply:
@sigewuzhere The link can work here. Good idea! The link can point to https://configcat.com/support/. What do you think?